packages/react-native-renderer/src/ReactNativeHostConfig.js

```diff
@@ -454,15 +454,29 @@ export function resetTextContent(instance: Instance): void {
 }
 
 export function hideInstance(instance: Instance): void {
-  throw new Error('Not yet implemented.');
+  const viewConfig = instance.viewConfig;
+  UIManager.updateView(
+    instance._nativeTag,
+    viewConfig.uiViewClassName,
+    {display: 'none'},
+  );
 }
 
 export function hideTextInstance(textInstance: TextInstance): void {
   throw new Error('Not yet implemented.');
```

Comment:
Can this happen? E.g. returning a string from render?

Reply:
if you do

```
<Text><Suspense>{string}</Suspense></Text>
```

I think so. it seems weird, I don't know if this is complicated, and we never do this right now so I opted to skip it

Reply:
I believe these are just like any other shadow node inside text from the native's perspective so I think we could send the display: none property here too.

cc @shergin

Reply:
I believe currently the only most outer `<Text>` supports `display: none`, but I think that's incorrect behavior (and I will fix it).

Reply:
I believe in the DOM version we actually just set the string to empty string `""` while hidden and then reset it to its current value.

Seems like we could do the same for RN without needing to fix the native side.